frontend/app-development/features/dataModelling/SchemaEditorWithToolbar/SchemaGenerationErrorsPanel.tsx

```diff
@@ -0,0 +1,57 @@
+import classes from './SchemaGenerationErrorsPanel.module.css';
+import React from 'react';
+import { Alert, Button, ErrorMessage, Paragraph } from '@digdir/design-system-react';
+import { Trans, useTranslation } from 'react-i18next';
+import { XMarkIcon } from '@navikt/aksel-icons';
+
+export interface SchemaGenerationErrorsPanelProps {
+  onCloseErrorsPanel: () => void;
+  schemaGenerationErrorMessages: string[];
+}
+
+export const SchemaGenerationErrorsPanel = ({
+  onCloseErrorsPanel,
+  schemaGenerationErrorMessages,
+}: SchemaGenerationErrorsPanelProps) => {
+  const { t } = useTranslation();
+
+  return (
+    <Alert severity='danger'>
+      <div className={classes.errorPanel}>
+        <div>
+          <Paragraph>{t('api_errors.DM_01')}</Paragraph>
+          <ul>
+            {schemaGenerationErrorMessages?.map((errorMessage, index) => {
+              return (
+                <li key={index}>
+                  <ErrorMessage>
+                    {errorMessage.includes(
+                      'member names cannot be the same as their enclosing type',
+                    ) ? (
```

Comment:
Should we create a new method named `isKnownError(errorMessage)` which implements this detail? Then it's easier to know that we do this check because we know about the error. 

```suggestion
                    {isKnownError(errorMessage) ? (
```